Makefile

```
@@ -93,11 +91,7 @@ build-plain:
 
 .PHONY: build-image
 build-image:
-	$(CONTAINER_RUNTIME) build -t $(IMAGE_HOST)/$(IMAGE):$(TAG) -f $(DOCKERFILE) .
-
-.PHONY: push-image 
-push-image:
-	$(CONTAINER_RUNTIME) push $(IMAGE_HOST)/$(IMAGE):$(TAG)
+	ko publish -P ./cmd/manager
```

The PR [Add trimpath Go build flag #565](https://github.com/shipwright-io/build/pull/565) has added some go flags. Another one is in review: [Tweak build operator build settings #569](https://github.com/shipwright-io/build/pull/569/files). I think there is an environment variable that can be used, [`GOFLAGS`](https://github.com/google/ko#how-can-i-set-ldflags). I guess changing this line to

`GOFLAGS=$(GO_FLAGS) ko publish -P ./cmd/manager`

might be needed.

---

Good point. `ko` always adds trimpath if the version of Go supports it: https://github.com/google/ko/blob/v0.8.0/pkg/build/gobuild.go#L317

We can add any other ldflags you'd like, as you said, using GOFLAGS.

---

Can we use the `GO_FLAGS` that are already in the file ?

```suggestion
	GOFLAGS=$(GO_FLAGS) ko publish -P ./cmd/manager
```